src/ServiceControl.Monitoring/ServiceControl.Monitoring.csproj

```diff
@@ -12,11 +12,26 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>ServiceControl.Monitoring</RootNamespace>
     <AssemblyName>ServiceControl.Monitoring</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
```

Comment:
Well, this is a stand-alone service so I guess we can chose whatever we want. What value makes sense these days?